dist/dfu.js

```diff
@@ -382,7 +382,8 @@
                         case OPCODE.VALIDATE_FIRMWARE:
                             log('complete, reset...');
 
-                            controlChar.service.device.addEventListener("gattserverdisconnected", function() {
+                            server.device.addEventListener("gattserverdisconnected", function() {
```

Comment:
Yes, this is not implemented yet (commented out actually): https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/modules/bluetooth/BluetoothRemoteGATTCharacteristic.idl&q=bluetooth%20characteristic.idl&sq=package:chromium&type=cs&l=17